cuda/base/cusparse_bindings.hpp

```diff
@@ -166,15 +172,21 @@ GKO_BIND_CUSPARSE_TRANSPOSE64(ValueType, detail::not_implemented);
         const ValueType *OrigValA, const int32 *OrigRowPtrA,                 \
         const int32 *OrigColIndA, ValueType *TransValA, int32 *TransRowPtrA, \
         int32 *TransColIndA, cusparseAction_t copyValues,                    \
-        cusparseIndexBase_t idxBase) GKO_NOT_IMPLEMENTED
+        cusparseIndexBase_t idxBase) GKO_NOT_IMPLEMENTED;                    \
+    static_assert(true,                                                      \
+                  "This assert is used to counter the false positive extra " \
+                  "semi-colon warnings")
 
 #define GKO_BIND_CUSPARSE_CONJ_TRANSPOSE64(ValueType, CusparseName)          \
     inline void conj_transpose(                                              \
         cusparseHandle_t handle, size_type m, size_type n, size_type nnz,    \
         const ValueType *OrigValA, const int64 *OrigRowPtrA,                 \
         const int64 *OrigColIndA, ValueType *TransValA, int64 *TransRowPtrA, \
         int64 *TransColIndA, cusparseAction_t copyValues,                    \
-        cusparseIndexBase_t idxBase) GKO_NOT_IMPLEMENTED
+        cusparseIndexBase_t idxBase) GKO_NOT_IMPLEMENTED;                    \
+    static_assert(true,                                                      \
+                  "This assert is used to counter the false positive extra " \
+                  "semi-colon warnings")
```

Comment:
You don't need these `asserts` because you already have them in `GKO_NOT_IMPLEMENTED`.
Just the semicolons after `GKO_NOT_IMPLEMENTED` should be fine.

Reply:
Yep, sorry..

Reply:
Sorry, it is actually needed, because that is a macro in itself and is used later.

Reply:
In that case, it should be as it was before (without the assert and without the semicolon).
Sorry for the confusion.

Reply:
I think the way it is now is also fine, because ideally, you would like to replace the `GKO_NOT_IMPLEMENTED ` with an implementation and then you would need the asserts.

Reply:
That is true.